### PR TITLE
4.18.x+fslc

### PR DESCRIPTION
**Pull request by schnitzeltony**
Hi @otavio & @fabioestevam 

This PR contains
1. ARM imx_v6_v7_defconfig: remove CONFIG_USB_CHIPIDEA_ULPI and CONFIG_USB_ULPI_BUS
2. ARM imx_v6_v7_defconfig: add CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
3. Merge remote-tracking branch 'mainline-stable/linux-4.18.y' into 4.18.x+fslc
4. Add CONFIG_TMPFS_POSIX_ACL=y

Details:
1+2 same as for 4.17 sent yesterday
3. Update to 4.18.9 from git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
4. That did not make it from 4.17 -> 4.18

I'll send updates to meta-freescale once this is accepted/applied

### CODE REVIEW

**Comment by schnitzeltony** [comment]
Sorry for the noise but I just added the yocto menuconfig fixes [1]

[1] http://lists.openembedded.org/pipermail/openembedded-core/2018-September/155985.html

**Comment by otavio** [comment]
I pushed the v4.18.9 merge. Is it possible for you to rebase and push it again?

**Comment by schnitzeltony** [comment]
Done

File: scripts/kconfig/mconf-cfg.sh [hunk 1]
```diff
@@ -12,6 +12,14 @@ if [ "$CROSS_CURSES_LIB" != "" ]; then
...
 fi
 
+if [ "$CROSS_CURSES_LIB" != "" ]; then
+	echo libs=\'$CROSS_CURSES_LIB\'
+	if [ x"$CROSS_CURSES_INC" != x ]; then
+		echo cflags=\'$CROSS_CURSES_INC\'
+	fi
+	exit 0
+fi
+
```
Comment (otavio) [inline scripts/kconfig/mconf-cfg.sh:14]: This change is duplicated. Please drop it.

File: arch/arm/configs/imx_v6_v7_defconfig [hunk 2]
```diff
@@ -410,6 +410,8 @@ CONFIG_ZISOFS=y
 CONFIG_UDF_FS=m
 CONFIG_MSDOS_FS=m
 CONFIG_VFAT_FS=y
+CONFIG_TMPFS=y
+CONFIG_TMPFS_POSIX_ACL=y
```
Comment (otavio) [inline arch/arm/configs/imx_v6_v7_defconfig:414]: Change is right, the commit log must mention what it is changing.

Comment (schnitzeltony) [reply arch/arm/configs/imx_v6_v7_defconfig:414]: What is missing here?

**Pull request merged**